spec.bs

```diff
@@ -2328,7 +2330,7 @@ Each {{InterestGroupBiddingScriptRunnerGlobalScope}} has a
 : <dfn>bid</dfn>
 :: A [=generated bid=]
 : <dfn>priority</dfn>
-:: Null or a {{double}}
+:: Null or failure or a {{double}}

```

Comment:
Nit: add a comma. So, `Null, failure, or double`

Reply:
Done (and set the default)